src/main/java/menagerie/gui/screens/duplicates/DuplicatesScreen.java

```diff
@@ -196,6 +135,10 @@ protected void updateItem(Tag tag, boolean empty) {
     });
   }
 
+  private SimilarPair<MediaItem> getCurrentPair() {
```

Comment:
Why introduce this method if it's only used twice?

Reply:
good point, will remove it in a future PR 👍